cartography/intel/bitbucket/members.py

```diff
@@ -0,0 +1,68 @@
+import requests
+import logging
+from typing import Any
+from typing import Dict
+from typing import List
+from requests.exceptions import RequestException
+import neo4j
+from cartography.util import make_requests_url
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+logger = logging.getLogger(__name__)
+
+@timeit
+def get_workspace_members(access_token:str,workspace:str):
+    url = f"https://api.bitbucket.org/2.0/workspaces/{workspace}/members"
+    return make_requests_url(url,access_token)
+
+
+
+    
+def load_memebers_data(neo4j_session: neo4j.Session,members_data:List[Dict],common_job_parameters:Dict):
+    ingest_workspace="""
+    UNWIND $membersData as member
+    MERGE (mem:BitbucketMember{name: member.user.display_name})
```

Comment:
use unique id to load resources if not present then create with standard format

Reply:
uuid unique attribute  as id